exporter/jaeger/jaegergrpcexporter/config.go

```diff
@@ -12,7 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package jaegerexporter contains a specialized Jaeger exporter. Unlike client
```

Comment:
Why doc.go is removed?

Reply:
There won't be a jaegerexporter package anymore, each one of the protocols gets its own package, this PR creates one, two more to come. That said the package that I created doesn't have a comment... we are missing some tool to enforce that.

Reply:
Agree, I haven't found a tool in open source on Godoc check. For this new package please add a doc.go.